src/main/java/net/malisis/core/MalisisCore.java

```diff
@@ -62,7 +62,11 @@
 /**
  * The Class MalisisCore.
  */
-@Mod(modid = MalisisCore.modid, name = MalisisCore.modname, version = MalisisCore.version)
+@Mod(
+        modid = MalisisCore.modid,
+        name = MalisisCore.modname,
+        version = MalisisCore.version,
+        dependencies = "required-after:gtnhlib@0.0.10")
```

Comment:
```suggestion
        dependencies = "required-after:gtnhlib@[0.0.10,)")
```
Otherwise it'll be locked to version 0.0.10

Reply:
According to <https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html> which fml follows they should be equivalent

Reply:
Actually after checking the code it seems my version restriction does nothing

Reply:
Even if it did, I'd find `[0.0.10,)` more clear